_posts/2021-11-22-using-context-to-simplify-a-react-form.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: epic
+title: Using Context to Simplify a VERY Large React Form
+subtitle: How We Took Incremental Steps to Revamp the Artwork Form at Artsy
+date: 2021-11-22
+categories: [refactoring, react, context, typescript]
+author: [anna, laura]
+---
```

Comment:
If you want to add a comment thread, see https://github.com/artsy/artsy.github.io/#enabling-comments